webrender/src/mask_cache.rs

```diff
@@ -149,46 +148,39 @@ impl MaskCacheInfo {
                   gpu_cache: &mut GpuCache,
                   device_pixel_ratio: f32)
                   -> &MaskBounds {
-
         // Step[1] - compute the local bounds
         //TODO: move to initialization stage?
         if self.bounds.inner.is_none() {
             let mut local_rect = Some(LayerRect::new(LayerPoint::new(-MAX_CLIP, -MAX_CLIP),
                                                      LayerSize::new(2.0 * MAX_CLIP, 2.0 * MAX_CLIP)));
-            let mut local_inner: Option<LayerRect> = None;
+            let mut local_inner = local_rect;
             let mut has_clip_out = false;
             let has_border_clip = !self.border_corners.is_empty();
 
             for source in sources {
                 match *source {
-                    ClipSource::Complex(rect, radius, mode) => {
+                    ClipSource::Image(ref mask) => {
```

Comment:
now that we have 1:1 relationship between `local_rect`/`local_inner` modification and a `source` item, we may go fancy and use `fold()` here ;)